### DISCUSSION TITLE
Remove automatic chapter numbering from table of contents and titles

### DISCUSSION THREAD

**Discussion by slawrensen** [brechtm/rinohtype#357] [General]
Hello, can you assist me with removing the chapter numbering from the ToC and titles? I have been researching and attempting to modify findings to my needs, but have been unsuccessful. I understand I may need to use "number_format = none" in my stylesheet?

I consulted the below question but was not able to modify your answer for my needs. Thanks!

https://github.com/brechtm/rinohtype/issues/156

**Comment by brechtm** [comment]
Assuming you're building on top of the Sphinx style sheet, you need to only override the _number_format_ property for the _heading level 1_ style in your style sheet:
```
[STYLESHEET]
base = sphinx

[heading level 1]
number_format = none
```

Since the other headings in the Sphinx stylesheet inherit _number_format_ from _heading level 1_, these will also be unnumbered.
